README.md

```diff
@@ -120,10 +120,11 @@ docker build -t hmda-api .
 
 Then, run the docker image
 ```shell
-docker run -d -p "8080:8080" hmda-api
+docker run -d -p "8080:8080 -p 8082:8082" hmda-api
 ```
 
-The API will run on `$(docker-machine ip):8080`
+The Filing API will run on `$(docker-machine ip):8080`
+The Public API will run on `$(docker-machine ip):8082`
```

Comment:
Will add documentation as soon as the `institutions search` PR gets merged, since that document exists as part of that PR